frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -182,44 +219,104 @@ class DatasetImportView extends React.PureComponent<Props, State> {
       });
     }
 
+    let message;
+    let type = "info";
+    const didParsingTheSavedDataSourceJSONSucceed = savedDataSourceOnServer.status == null;
+    if (!didParsingTheSavedDataSourceJSONSucceed) {
+      // If the datasource-properties.json on the server is an invalid JSON.
+      message = (
+        <div>
+          The current datasource-properties.json on the server seems to be in an invalid JSON format
+          and webKnossos could not parse this file. The settings below are suggested by webKnossos
+          and should be adjusted. <br />
+          Be aware that webKnossos cannot guess properties like the voxel size or the largest
+          segment id. You must set them yourself.
+        </div>
+      );
+      type = "warning";
+    } else if (!isDatasourceJSONValid(savedDataSourceOnServer)) {
+      // If the datasource-properties.json on the server has invalid or missing properties but is a valid JSON.
+      message = (
+        <div>
+          The current datasource-properties.json on the server seems to have invalid or missing
+          properties. <br />
+          <a
+            href="#"
+            onClick={() =>
+              showJSONModal("Current and invalid datasource-properties.json", inferredDataSource)
```

Comment:
Doesn't this only show the inferred data source? The description says "Current and invalid datasource-properties.json", though.